machines/app.ts

```diff
@@ -61,6 +65,24 @@ export const appMachine = model.createMachine(
     },
     id: 'app',
     initial: 'init',
+    on: {
+      DECRYPT_ERROR: {
+        actions: [
+          () => {
+            console.log('decrypt error caught');
+          },
+          'setIsDecryptError',
```

Comment:
Can we extract this log action to a method / merge it with setIsDecryptError and name accordingly to avoid direct method in actions?

Reply:
I forgot to remove this log